gst/nnstreamer/elements/gsttensor_transform.c

```diff
@@ -2143,6 +2143,22 @@ gst_tensor_transform_transform_caps (GstBaseTransform * trans,
     structure = gst_caps_get_structure (caps, i);
     gst_tensors_config_from_structure (&in_config, structure);
 
+    if ((!gst_structure_has_field (structure, "format") ||
```

Comment:
Suggestion:
1. moving this before getting tensors-config (line 2144) `gst_tensors_config_from_structure (&in_config, structure);`
then we dont need to free in/out config.
2. fix mem leak
```
    GstCaps *result_aux;
    ....
    if (gst_structure_has_name (structure, NNS_MIMETYPE_TENSOR)) {
      result_aux = gst_tensor_caps_from_config (&out_config);
    } else {
      result_aux = gst_tensors_caps_from_config (&out_config);
```